shims/common/src/main/scala/io/glutenproject/GlutenConfig.scala

```diff
@@ -779,6 +781,16 @@ object GlutenConfig {
       .longConf
       .createWithDefault(100 * 1000 * 1000)
 
+  val COLUMNAR_FALLBACK_POLICY =
+    buildConf("spark.gluten.sql.columnar.fallback.policy")
+      .internal()
+      .doc("The fallback policy in gluten. By default, 'operator' means fallback each " +
+        "operator, 'stage' means fallback each stage, 'query' means fallback query")
```

Comment:
stage fallback only for AQE on, right ?

Reply:
right, the current impl replies AQE to check if it's fully supported